index.html

```diff
@@ -195,6 +201,7 @@ <h2 class="uppercase text-gray-700 text-sm">commercial examples</h2>
 							}
 						})
 					}
+					window.scrollTo(0,0);					 
```

Comment:
Methods `buildStyleLabel()` and `styleBox()` are fired for every tile (with `styleBox()` fired with every tile action), so basically you're scrolling to top about... 12240 times. ;)

If you want to scroll page, you can add `window.scrollTo(0,0)` at the end of `selectStyle()` method.

Reply:
Moved the method, any idea why your scroll would jump though? after select Style has finished?  it doesnt jump every time ,  hit every time, which is what confuses me. the only time i been able to concistently figure out one  tag will make the scroll jump all the way down is if I press "Kellerbier" with all the others i get proper scrolling. maybe  something when you build or organize im not sure